third_party/android-gmaps-addons/src/com/cocoahero/android/gmaps/addons/mapbox/MapBoxOfflineTileProvider.java

```diff
@@ -47,6 +47,18 @@ public MapBoxOfflineTileProvider(String pathToFile) {
     @Override
     public Tile getTile(int x, int y, int z) {
         Tile tile = NO_TILE;
+        String pathToFile = this.basepath + "/" + z + "-" + x + "-" + y + ".mbtiles";
+        File f = new File(pathToFile);
```

Comment:
We should limit the usage of single-variable names to standard usages. (e.g. i, j, k for counters, e for exceptions, etc.).

Please rename this to `file`

Reply:
done! b461845